client/src/app/core/ui-services/linenumbering.service.ts

```diff
@@ -138,7 +138,7 @@ export class LinenumberingService {
     private ignoreInsertedText = false;
 
     // A precompiled regular expression that looks for line number nodes in a HTML string
-    private getLineNumberRangeRegexp = RegExp(/<span[^>]+data\-line\-number=\"(\d+)\"/, 'g');
+    private getLineNumberRangeRegexp = RegExp(/<*data\-line\-number=\"(\d+)\"/, 'g');
```

Comment:
@CatoTH 
change recos were not rendered becuase `getLineNumberRange` returned 
```
{
  from: null,
  to: null
};
```
because the content of change recos was `SPAN` instead of `span`

Reply:
Oh, ok. But `<*` looks a bit strange - it means " a random number of <'s, or none". If you want to match any tag at all, you'd want to use `<[^>]*`, no? Or just making the regexp case-insensitive by adding an `i` to the flags (`gi`) would allow a `SPAN`, too...

Reply:
yes gi looks better. I always need to re-learn regex whenever I use it